backend/app/api/graph.py

```
@@ -369,13 +379,13 @@ def build_graph():
         # 更新项目状态
         project.status = ProjectStatus.GRAPH_BUILDING
         project.graph_build_task_id = task_id
-        ProjectManager.save_project(project)
+        await ProjectManager.save_project(project)
         
         # Capture locale before spawning background thread
         current_locale = get_locale()
 
         # 启动后台任务
-        def build_task():
+        async def build_task():
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `build_task`

Add return type annotation: `None`

(ANN202)